Formula/nef.rb

```diff
@@ -21,6 +21,6 @@ def install
     system "#{bin}/nef", "markdown",
            "--project", "#{share}/tests/Documentation.app",
            "--output", "#{testpath}/nef"
-    assert_path_exist "#{testpath}/nef/library/apis.md", :exist?
+    assert_path_exists "#{testpath}/nef/library/apis.md", :exist?
```

Comment:
Do we still need that `:exist?` argument at the end?

Reply:
Seeing as the others don't have it, I'd think not.

Reply:
What I was thinking.